mypyc/genops.py

```diff
@@ -284,9 +284,12 @@ def compute_vtable(cls: ClassIR) -> None:
     all_traits = [t for t in cls.mro if t.is_trait]
 
     for t in [cls] + cls.traits:
-        for fn in itertools.chain(t.properties.values(), t.methods.values()):
+        # prop_defs = list(itertools.chain(*t.properties.values()))
+        # for fn in itertools.chain(prop_defs, t.methods.values()):
+        for fn in itertools.chain(t.methods.values()):
+            # The function may be None in the case of an undefined property setter
             # TODO: don't generate a new entry when we overload without changing the type
-            if fn == cls.get_method(fn.name):
+            if fn and fn == cls.get_method(fn.name):
```

Comment:
I think the `fn` check can be dropped now, along with the comment explaining it.